scripts/validation-suite.sh

```
@@ -0,0 +1,357 @@
+        log_info "SREP Ticket:  $TICKET"
+        log_info "Cluster:      $(oc whoami --show-server 2>/dev/null || echo 'Not connected')"
+        log_info "Start Time:   $(date -u +%Y-%m-%dT%H:%M:%SZ)"
+        log_info ""
+    fi
+
+    # Check 1: Deployment Health
+    if ! should_skip "deployment"; then
+        log_header "1. Deployment Health Check"
+
+        if [[ "$JSON_OUTPUT" == "true" ]]; then
+            DEPLOYMENT_RESULT=$("$SCRIPT_DIR/validate-deployment.sh" $COMMON_ARGS -j 2>&1)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)